SkyhookGame/Assets/Scripts/Institutions/Base/Institution.cs

```diff
@@ -2,12 +2,21 @@
 
 public abstract class Institution : MonoBehaviour
 {
-    public int Level { get; private set; }
+    public int Level { get; private set; } = 1;
 
     public InstitutionType InstitutionType { get; private set; }
 
+    private void Awake()
```

Comment:
is that method shared between inherited classes if its not specified as virtual or smth?

Reply:
It is shared between inherited classes, yes. Although it might be a good idea to mark it as virtual already, so that child classes can override the Awake method if it necessary in the future